app/controllers/users_controller.rb

```diff
@@ -18,8 +25,12 @@ def accept_terms
 
 private
 
+  def user
+    @user ||= User.find(current_user["user_id"]).first
```

Comment:
Similar to my comment about reifying a user in `SessionsController#next_state`, do we really need to do this here? At first glance it looks like we should already have the user object from when we created the object.